differential_pressure_sensor/README.md

```diff
@@ -62,8 +62,8 @@ Next, we launch a PnP node-ID allocator available in Yakut (PX4 also implements
 
 ```bash
 export UAVCAN__CAN__IFACE="socketcan:vcan0"
-export UAVCAN__NODE__ID=127                 # This node-ID is for Yakut.
-yakut monitor -P ~/allocation_table.db
+export UAVCAN__NODE__ID=127                     # This node-ID is for Yakut.
+y mon --plug-and-play ~/allocation_table.db
```

Comment:
Where was this abbreviation aliased? Might be better to keep it expanded considering this is a README?

Reply:
Hmm I guess it's used a lot and is fine, so never mind.

Reply:
"Y A K U T" is verbose -- who has the time to type all these letters anyway?